mesop/tests/e2e/focus_component_test.ts

```diff
@@ -51,7 +51,7 @@ test('focus select', async ({page}) => {
   await page.goto('/focus_component');
   await page.locator('//mat-select[@id="mat-select-0"]').click();
   await page.getByRole('option', {name: 'Select'}).click();
-  await expectIsFocused(page, 'mat-select[id="mat-select-2"]');
+  await expectIsFocused(page, 'mat-select[id="mat-select-1"]');
```

Comment:
Why did this change to 1?

Reply:
I don't know. It seems weird it _used_ to be mat-select-2 because there's only two selects on the page (note: it's 0-indexed) so mat-select-1 seems right to me.